<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1762</width>
    <height>775</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>47</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>–ß–∞—Ç</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="chatEdit">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>88</y>
     <width>1461</width>
     <height>571</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1600</x>
     <y>700</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>–≠–º–æ–¥–∑–∏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>740</y>
     <width>80</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>üòé</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1757</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>–ê–¥—Ä–µ—Å —Å–µ—Ä–≤–µ—Ä–∞</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="serverIpEdit">
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>–ü–æ—Ä—Ç —Å–µ—Ä–≤–µ—Ä–∞</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="serverPortEdit">
      <property name="text">
       <string>40000</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>–í–∞—à –Ω–∏–∫</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nickEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="connectToServerButton">
      <property name="text">
       <string>–ü–æ–¥–∫–ª—é—á–∏—Ç—å—Å—è</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="sendMessageButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1600</x>
     <y>660</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sendMessageEdit">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>660</y>
     <width>1561</width>
     <height>112</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="nickList">
   <property name="geometry">
    <rect>
     <x>1470</x>
     <y>90</y>
     <width>271</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>50</y>
     <width>377</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string> –ü–æ–¥–∫–ª—é—á–µ–Ω–Ω—ã–µ –∫–ª–∏–µ–Ω—Ç—ã</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
